ta/ta.mk

```diff
@@ -121,7 +121,7 @@ $(conf-mk-file-export): $(conf-mk-file)
 	$(q)echo sm := $(sm-$(@)) > $@
 	$(q)echo sm-$(sm-$(@)) := y >> $@
```

Comment:
same in the 2 lines above. sm-$(@) may not extend to `$(out-dir)/export-$(sm)/mk/conf.mk` if `$(out-dir)` starts with `./`